readMe.md

```diff
@@ -83,10 +86,19 @@ These secrets will be added as shown below-
 
 
 ### 4. Setup and Define Triggers
+
+### Events that trigger workflow
+Github workflows are triggered based on events specified inside workflows. These events can be from inside the github repo like a push commit or can be from outside like a webhook([repository-dispatch](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#repository_dispatch)).
+Details can be found [here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows).
+
 #### Setup Trigger
 
-We have precreated workflow [setup_acr_trigger](/.github/workflows/setup_acr_trigger.yml) that enables our GitHub Repo to listen to events from this ACR. 
-User needs to set the following environment variables in this workflow-
+We have precreated workflow [setup_acr_trigger](/.github/workflows/setup_acr_trigger.yml) that deploys resources required for setup.
```

Comment:
"Precreated" is not a word in english. Change it to sample workflow instead of precreated workflow